PFCalEE/include/EventAction.hh

```diff
@@ -53,7 +53,7 @@ private:
   std::vector<SamplingSection> *detector_;
   G4int     evtNb_,printModulo;
 
-  HGCSSGeometryConversion geomConv_;
+  HGCSSGeometryConversion* geomConv_;
```

Comment:
is there a reason to make this a pointer ? I personnally prefer the object directly given we normally want only one instance of this class. If we use the pointer we should also add explicitely the "Delete" in the destructor....